types/node-media-server/node-media-server-tests.ts

```diff
@@ -0,0 +1,21 @@
+import NodeMediaServer from 'node-media-server';
```

Comment:
Should this be `import NodeMediaServer = require('node-media-server');`?  Then you wouldn't need `esModuleInterop`.